fieldservice_logbook/models/fsm_equipment_logbook_mixin.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+
+
+class FsmEquipmentLogbookMixin(models.AbstractModel):
+    _name = "fsm.equipment.logbook.mixin"
+    _description = "Fsm Equipment Logbook Mixin"
+
+    equipment_logs_count = fields.Integer(compute="_compute_equipment_logs")
+    equipment_logs_ids = fields.Many2many(
+        "fsm.equipment.logbook", compute="_compute_equipment_logs"
+    )
+
+    def _read_group_logbook_data(self):
```

Comment:
```suggestion
    def _read_group_logbook_data(self, logbook):
```

use the expected signature (and perhaps a nice docstring explaining what this method should do)